scenarios/kubernetes_e2e_test.py

```diff
@@ -473,6 +482,25 @@ def test_aws(self):
         self.assertEqual(
             self.envs['JENKINS_AWS_CREDENTIALS_FILE'], temp.name)
 
+    def test_use_shared_build(self):
+        args = kubernetes_e2e.parse_args([
+            '--use-shared-build'
+        ])
+        # normal path
+        with Stub(kubernetes_e2e, 'check_env', self.fake_check_env):
+            with Stub(kubernetes_e2e, 'read_gcs_path', always_kubernetes):
+                kubernetes_e2e.main(args)
```

Comment:
oh, here in the test, I would like to see the gcs path pass into `read_gcs_path` is correct, both with normal build and bazel build.

Reply:
done.